backend/server/main.py

```diff
@@ -60,3 +60,29 @@ def protected_route(user_id: str = Depends(auth)):
 @app.get("/test/admin")
 def admin_route(user_id: str = Depends(admin)):
     return {"message": "This is an admin route!!!! You are an admin."}
+
+# ---------------------------------------------------------------------------- #
+#                                Health Check                                  #
+# ---------------------------------------------------------------------------- #
+
+@app.get("/health", status_code=200)
+async def health_check(response: Response):
+    try:
+        # Perform a quick database query to check connectivity
+        await DB.queryOne("SELECT 1")
+        
+        # Set Cache-Control header to prevent caching
+        response.headers["Cache-Control"] = "no-store, no-cache, must-revalidate, max-age=0"
+        
+        # Return a JSON response indicating the backend is healthy
+        return {"status": "healthy", "message": "Backend and database are operational"}
```

Comment:
```suggestion
        return {"backend": True, "api": True}
```
To match the API contract:
https://jaxcksn.github.io/CS3365-MBS/#operation/health-check